ivy/functional/frontends/torch/Tensor.py

```diff
@@ -128,6 +128,10 @@ def _to_with_device(
     def arctan(self, *, out=None):
         return torch_frontend.arctan(self, out=out)
```

Comment:
The test is still failing, but it's because `arctan` is not defined correctly (not your fault obviously). Could you change this line to `return torch_frontend.atan(self, out=out)`? In torch `arctan` is an alias for `atan`.